docs/source/share_kedro_viz.md

```diff
@@ -23,7 +23,26 @@ When prompted for a project name, you can enter anything, but we will assume `Sp
 
 When your project is ready, navigate to the root directory of the project.
 
-## Update and install the dependencies
+## Platform-Agnostic sharing with Kedro-viz 
+
+Starting from Kedro-Viz version 7.1.0, we introduced the `kedro viz build` command. When a developer runs this command from the command line interface (CLI), it will create a `build` folder within their Kedro project. This build folder contains the entire static Kedro-Viz app package, which users can manually publish to any hosting platform of their choice.
+
+Some suggested platforms that have been explored with `kedro viz build` include:
+* For static website hosting platforms like GitHub Pages:
+    * Follow the steps linked [here](https://docs.github.com/en/pages/quickstart) to create a Git repository that supports GitHub Pages.
+    * Once that's done, simply push the contents of the `build` folder to this new repository.
+    * Your site will be available at the following URL: `http://<username>.github.io`
+* For cloud providers such as AWS, Azure, and GCP:
+    * After setting up your bucket and enabling static hosting, manually update the contents of the `build` folder in the bucket.
```

Comment:
After setting up your bucket and enabling static hosting, manually update the contents of the `build` folder in the bucket. -> **After creating a bucket and configuring it for static website hosting, copy the contents of the `build` folder to the bucket manually**